getting-started-tutorial/socless-tutorial/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "socless-tutorial",
+  "config": {
+    "aws_profile": "default",
+    "dev": "--stage dev --region us-west-2",
+    "prod": "--stage prod --region us-east-1",
+    "sandbox": "--stage sandbox --region us-west-1"
+  },
+  "version": "1.0.0",
+  "scripts": {
+    "test": "serverless --version",
+    "dev": "serverless deploy $npm_package_config_dev --aws-profile $npm_package_config_aws_profile --verbose",
+    "prod": "serverless deploy $npm_package_config_prod --aws-profile $npm_package_config_aws_profile --verbose",
+    "sandbox": "serverless deploy $npm_package_config_sandbox --aws-profile $npm_package_config_aws_profile --verbose"
+  },
+  "license": "ISC",
+  "dependencies": {
+    "serverless": "^1.48.2",
```

Comment:
We should change this to `"serverless": "^1.50.0",` because current serverless version is broken due to outdated `graceful-fs` dependency. Removing the `^` will prevent auto updates via back-to-back npm install